### DISCUSSION TITLE
Integration between Vercel & Cloudflare

### DISCUSSION THREAD

**Discussion by 258CriptonBaloi** [vercel/community#2396] [Help] [answered]
### Summary

Hello everybody! Hope you guys are having a great day.

I need your help. I just put my site in production and it's working perfectly. However, I am having issues with my domain which is being managed by Cloudflare and is not working properly. Could you help me to solve this problem?

### Additional information

the domain is criptontech.com

**Comment by amyegan** [comment, marked as answer]
Hi @258CriptonBaloi! DNS records changes can sometimes take a while to propagate. Does the custom domain verified as "Valid Configuration" in your project settings? If not, the first thing I recommend is to check DNS propagation status. You can find the steps here: [Why is my Vercel domain not verified?](https://vercel.com/guides/why-is-my-vercel-domain-unverified#cname-record-changes)

If something else is going wrong, please describe exactly what problem you're seeing. Screenshots may be helpful for this.

**Reply by 258CriptonBaloi** [reply]
Hello @amyegan, I would like to inform you that my domain is already verified and to prove it, I will share a screenshot. Also, I put the same Vercel DNS settings on Cloudflare and now my website is working perfectly. Thank you very much for your attention and I wish you a great day!

![Captura de tela 2023-05-11 083838](https://github.com/vercel/community/assets/72799525/78e62019-0519-4203-ac83-30565e74e629)
